tests/README.md

```diff
@@ -0,0 +1,90 @@
+MetalK8s is an infrastructure product, that rely on ansible so framework in python is somewhat “obvious”.
+Pytest is quite famous right famous.
+
+Test to be run could be splitted in different category:
+
+1. Test that run before installation on any machine
+    - test of os-check (centos check and kernel)
+    - test first installation of component that doesn't need kubernetes (node_exporter)
+    - test of the complete installation it-self
+
+2. Test that run after the installation of kubernetes only (not the entire set of MetalK8s):
+    - first installation of elasticsearch
+
+3. Test that could be run at each point of time after MetalK8s installation (non-destructive):
+    - test of prometheus monitoring
+
+4. Test that meant to be run after installation and before production (conformance test)
+    - test kubernetes object creation
+
+5. Test that need to run on non-production environnement (CI test)
```

Comment:
Include K8s Conformance suite.

Reply:
Indeed

Reply:
Considering the time to run conformance, we might not be able to run it at each commit, or merge. May be that's something we can have at each "release" (including alpha or rc to see problem before needing the release). IHM, we do not need to integrate this in the pytest framework (and it takes 2 hours, no parallelism possible), but keeping the sonobuoy runner that we could run directly from our CI

Reply:
Fair enough. Kind of a 'nightly' (and indeed, pre-release).